src/convert.cpp

```diff
@@ -274,8 +274,8 @@ class Converter {
 
   // DATA
   static const Conversion conversion_[];  //<! Conversion rules
-  bool erase_;
-  bool overwrite_;
+  bool erase_{};
```

Comment:
Why did you remove the value `false` here? 

By the way, you could also initialize here the pointers to `nullptr`

Reply:
Constructors already handle that.

Reply:
What I meant is that before we had in the constructors this:

```cpp
Converter::Converter(ExifData& exifData, XmpData& xmpData) :
    erase_(false),
    overwrite_(true),
```

And now you moved the initialization to the .h (this for me is fine) where the variables are declared:

```cpp
  bool erase_{};
  bool overwrite_{true};
```

However, you are initializing with an explicit value only `overwrite_` and not `erase_`. Did you forget to assign the `false` value for erase?

Reply:
No. {} and {false} are equivalent. The former is shorter

Reply:
I do not think so. I have been bitten by such assumptions already many times. Depending on compiler, compiler version, compilation mode (Release or Debug) , platforms, etc. you can get different values when you are not explicit.

Even in the case this changed with latest standards, it is an obscure implementation detail. It is much better to be explicit about your intentions.

Reply:
Thanks ;)